velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -456,5 +456,56 @@ TEST_F(DateTimeFunctionsTest, quarterDate) {
   EXPECT_EQ(3, quarter("1954-08-08"));
 }
 
+TEST_F(DateTimeFunctionsTest, getTimestamp) {
+  const auto getTimestamp = [&](const std::optional<StringView>& dateString,
+                                const std::string& format) {
+    return evaluateOnce<Timestamp>(
+        fmt::format("get_timestamp(c0, '{}')", format), dateString);
+  };
+  const auto testGetTimestamp = [&](const std::optional<StringView>& dateString,
+                                    const std::string& format,
+                                    const StringView& expect) {
+    EXPECT_EQ(
```

Comment:
better move EXPEC_EQ to the caller so that error message has the right line number

Reply:
The code may be concise if we keep EXPECT_EQ in this lambda, as each test case needs `fromTimestampString` to convert string to expected timestamp. Right?

Reply:
It is strange that some code uses testGetTimestamp and some uses EXPECT_EQ directly. Using EXPECT_EQ provides better error messages pointing out exactly which test case failed. Consider changing getTimestamp to return formatted timestamp and use EXPECT_EQ directly.

Reply:
Makes sense. Just updated with Timestamp::toString() used to get string result for comparison. Thanks!